NuGet.Samples.Interop/NuGet.Samples.Interop.targets

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- See https://github.com/NuGet/Home/issues/2365 for more info.
+       The decision whether an assembly needs to be linked or referenced belongs to the package author.
+       -->
+  <Target Name="LinkEmbeddableAssemblies" AfterTargets="ResolveReferences" BeforeTargets="FindReferenceAssembliesForReferences">
+    <ItemGroup>
+      <ReferencePath Condition=" '%(FileName)' == 'NuGet.Samples.Interop' ">
```

Comment:
This looks good, are there any other conditions that can be checked? Does the reference contain the id of the package in the metadata?

Reply:
@emgarten 
Great point. 
I have added a check for ReferencePath.NuGetPackageId....that should make it as unique as it gets. 

There are additional conditions one would do if they have conflicts but I don't expect such scenarios.